src/main/webapp/main.js

```diff
@@ -1,9 +1,9 @@
 Vue.component("photo-copyright", {
   template: `
     <div id="source">
+      This web page uses images from 
       <a href="https://www.freepik.com/"> 
-        This web page uses images from freepik.com
-      </a>
+        freepik.com</a> and data from <a href="https://en.wikibooks.org/wiki/Cookbook:Table_of_Contents">wikibooks.org</a>
```

Comment:
Nit: formatting's a bit off.
You can try to follow the previously existing style:
```
<open_tag>
  content
</close_tag>
more content
```